atd-etl/app/process/helpers_import.py

```diff
@@ -122,10 +145,22 @@ def generate_gql(line, fieldnames, file_type):
         fields = generate_fields_with_filters(line=line,
                                               fieldnames=fieldnames,
                                               filters=filters)
-
-        template = generate_template(name=query_name,
-                                     function=function_name,
-                                     fields=fields)
+        # Clean field list
+        template_fields = [field.lower() for field in fieldnames if field.lower() not in filters[0][1]]
+        # Generate Template
+        template = generate_template(
+            name=query_name,
+            function=function_name,
+            fields=fields,
+            fieldnames=template_fields,
+            upsert=(file_type != "crash"),
```

Comment:
How are upserts for crash records handled? Or are they right now?

Reply:
No upserts for crashes at the moment, we are going to assume the crash change needs to be reviewed by a human.

There are plains for having automated upserts if the crash has not been edited by a human, but not within the scope of this issue.

Reply:
Awesome, thanks!